### DISCUSSION TITLE
Zoom command with liner FOV does not work when using archive mode using HERO 7 BLACK.

### DISCUSSION THREAD

**Discussion by Cayrow** [gopro/labs#821] [General]
I cannot make start archive mode with a zoom already configured. I can scan this command `mVr27p60fL99e0`  out of archive mode and it zooms in. However when using `!EmVdVq1mVr27p60fL99e0!MARCH=1"Archive Mode\nEnabled\nShutting-down"!O` instead of starting archive mode zoomed it starts it in wide mode without any zoom. Is there any way around this using a delayed command or something similar?

**Comment by dnewman-gpsw** [comment]
Moved to discussions as this is not strictly a broken feature on Labs. HERO7 does not remember a zoom level between boots, Labs inherits that limitation from the main firmware.  Unfortunately you can't use a boot command with Archive mode to fix that, as archive uses Quick Capture, which starts before the scripting from a boot command.
